script/DeployMorphoTokenBase.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+pragma solidity ^0.8.0;
+
+import "../lib/forge-std/src/Script.sol";
+import "../lib/forge-std/src/console.sol";
+
+import {MorphoTokenOptimism} from "../src/MorphoTokenOptimism.sol";
+import {ERC1967Proxy} from
+    "../lib/openzeppelin-contracts-upgradeable/lib/openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
+
+contract DeployMorphoTokenBase is Script {
+    address public constant MORPHO_DAO = 0xcBa28b38103307Ec8dA98377ffF9816C164f9AFa;
+    address public constant REMOTE_TOKEN = 0x12Ec7dF395E8B974537006Cf3DF8b8fFE5C0D41C;
```

Comment:
Maybe fork ethereum and sanity check this address (eg name&symbol) as part of the script.

Reply:
solved by the splitting